admin/app/components/solidus_admin/ui/forms/address/component.rb

```diff
@@ -1,13 +1,15 @@
 # frozen_string_literal: true
 
 class SolidusAdmin::UI::Forms::Address::Component < SolidusAdmin::BaseComponent
-  def initialize(form:, disabled: false)
+  def initialize(form:, address:, disabled: false, html_attributes: {})
```

Comment:
Also for consistency with other components I suggest turning `html_attributes:` into `**attributes`.

Reply:
To be honest, that option isn't even used 😅 It's a typo from an old implementation.